packages/neon_framework/pubspec.yaml

```diff
@@ -33,7 +32,12 @@ dependencies:
     sdk: flutter
   flutter_bloc: ^8.1.6
   flutter_blurhash: ^0.8.2
-  flutter_local_notifications: ^17.0.0
+  # https://github.com/MaikuB/flutter_local_notifications/pull/2418
+  flutter_local_notifications:
+    git:
+      url: https://github.com/MaikuB/flutter_local_notifications
+      ref: d9789d8496545ca2c5a42b78b9f49dc8efe2929b
+      path: flutter_local_notifications
```

Comment:
I had to fix this myself to allow mocking the plugin in tests. From the repository git activity it looks like we might get a new release soon so we could just wait for that if we don't want to depend on this single git commit.

Reply:
No activity since, so we just have to stay on this commit for now.